sunpy/timeseries/timeseriesbase.py

```diff
@@ -573,6 +573,8 @@ def to_array(self, **kwargs):
         `~numpy.ndarray`
             If the data is heterogeneous and contains booleans or objects, the result will be of ``dtype=object``.
         """
+        if pd.__version__ < "0.24.0":
```

Comment:
I don't think this version check will work, you probably need to use LooseVersion or whatever the "correct" way of doing version checking is.

Reply:
I feel like you can do basic string checking like that. It did seem to work locally when I installed pandas 0.23